lib/src/test_utils.rs

```diff
@@ -0,0 +1,32 @@
+use std::env;
+use std::process::Command;
+use std::str;
+use std::sync::Once;
+
+pub const FUNNEL_HOST: &str = "http://localhost";
+pub const FUNNEL_PORT: u16 = 8000;
+pub static INIT: Once = Once::new();
+
+pub fn setup() {
+    INIT.call_once(|| {
+        env::set_var("RUST_LOG", "debug");
+        env_logger::init();
+    });
+}
+
+pub async fn ensure_funnel_running() -> String {
+    let output = Command::new("sh")
+        .arg("-c")
+        .arg("ps aux | grep '[f]unnel server run'")
+        .output()
+        .expect("Failed to execute command");
+
+    let output_str = str::from_utf8(&output.stdout).unwrap();
+
+    if output_str.is_empty() {
+        panic!("Funnel is not running.");
+    }
+
+    let funnel_url = format!("{}:{}", FUNNEL_HOST, FUNNEL_PORT);
+    funnel_url
+}
```

Comment:
I think there should be no mention of Funnel in `lib`. I am not really comfortable with any tight coupling of code to specific implementations. It's (maybe) okay to use it for testing (even though it's not a reference implementation, and so tests against Funnel don't necessarily tell us much, one way or the other), but I think it that should be done strictly _outside_ of the main codebase.

Reply:
I have created a issue regarding this point at https://github.com/elixir-cloud-aai/ga4gh-sdk/issues/15. We can further do the necessary changes in the near future